packages/dht/src/connection/websocket/WebsocketConnector.ts

```diff
@@ -208,10 +216,11 @@ export class WebsocketConnector {
                     throw new Err.ConnectionFailed('ConnectivityChecker is destroyed')
```

Comment:
Not necessarily in this PR but looks like we do re-tries even when `this.abortController.signal.aborted` is true? Should this be checked in the catch-block to avoid unnecessary waiting and reattempts?